### ISSUE TITLE
Cannot Start Practice Mode after Get5 Scrim

### ISSUE THREAD

**Issue by aIexb** [splewis/get5#579]
If making a feature request, you should delete all the pre-filled text here.
If reporting a bug, fill in the following sections:

### Expected behavior

Start server in practice mode, start get5 scrim and disable practice mode. Once scrim is over be able to start practice mode again.

### Actual behavior
I can start practice mode and  start the get5 scrim, but once I end the get5 scrim, it says practice mode cannot be started.

### Steps to reproduce

Please note that "latest" is NOT a version! See https://github.com/splewis/get5/wiki/Debugging-help#finding-get5-version if you need help finding these. Adding the output file produced by ``get5_debuginfo`` is extremely helpful.

- Plugin version: 0.7.2-dev+486
- Sourcemod version: 1.10.0-git6497
- Steps to reproduce (please be specific): 

1. Join server
2. Start practice mode 
3. Start get5 scrim.
4. Disable practice mode.
5. get5 scrim starts once everyone is ready.
6. scrim ends
7. try starting practice mode and cannot.

**Comment by PhlexPlexico** [comment]
I think this can be adjusted with the config files. You may have to do the following, from #553:

- `sm_practicemode_can_be_started "0"` in your `get5/live.cfg`
- `sm_practicemode_autostart "0"` in your `get5/knife.cfg` and `get5/warmup.cfg` (just in case).
- Create another CFG, or if you have a CFG defined in `get5_autoload_config`, place `sm_practicemode_can_be_started "1"` inside that CFG, as well as `sm_practicemode_autostart "1"`

I believe this should fix the issues with the practicemode not starting, all with it being automated.

We haven't heard anything back for almost two years, so I'm going to close this as it seems like a duplicate of the aforementioned issue. 

Plus I think at some point there was an issue with this in the Discord with practice mode and money, and a reinstall of the server seemed to have rectified this issue as well. If there's still any issues with this, feel free to re-open, or comment in the other ticket :)

Thanks!
